conf/embl_hd.config

```diff
@@ -0,0 +1,46 @@
+params {
+    config_profile_contact     = "gitlab-data-sciences-sci-workflows-nextflow-config-6076-issue-@embl.de"
+    config_profile_description = "The EMBL Heidelberg HPC cluster profile"
+    config_profile_name        = 'EMBL_HD'
+    config_profile_url         = 'https://git.embl.de/data-sciences/sci-workflows/nextflow-config'
+}
+
+apptainer {
+    enabled = true
+    autoMounts = true
+    docker.enabled = false
+    pullTimeout = "3 hours"                // the default is 20 minutes and fails with large images
+    envWhitelist = 'CUDA_VISIBLE_DEVICES'  // allow the bounding of GPU visible device variable into the containers
+}
+
+process {
+    scratch = '$SCRATCHDIR'
```

Comment:
This may fail some resolving cases. I suggest instead trying using the `System.setenv` approach as done in the VS_UGHENT config: https://github.com/nf-core/configs/blob/e53bf51bd486452aa49c8b1b9024df44035e6cb6/conf/vsc_ugent.config#L17-L20

Reply:
ah.. didnt know that - made that change so